Assets/ModScripts/ComponentInfo.cs

```diff
@@ -6,48 +6,95 @@
+    public static readonly string[] MorseList = { "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z" };
 
-    //For converting adjacent arrow colors into their associated slider colors. Colors within string pairs are ordered alphabetically: Blue, Green, Red, Yellow
-    readonly Dictionary<string, int> SliderColors = new Dictionary<string, int>
+    // Dictionaries
+    // For converting adjacent arrow colors into their associated slider colors. Colors within string pairs must be ordered alphabetically: Blue, Green, Red, Yellow
+    static readonly Dictionary<string, int> SliderColors = new Dictionary<string, int>
     {
-        { "00", 0 }, //Blue
-        { "11", 1 }, //Green
-        { "22", 2 }, //Red
-        { "33", 3 }, //Yellow
-        { "01", 4 }, //Cyan
-        { "12", 5 }, //Golden Yellow
-        { "02", 6 }, //Magenta
-        { "23", 7 }, //Orange
-        { "03", 8 }, //Grey
-        { "13", 9 }, //White
+        { ((int)ArrowColors.Blue).ToString() +   ((int)ArrowColors.Blue).ToString(),   (int)ArrowColors.Blue },
+        { ((int)ArrowColors.Green).ToString() +  ((int)ArrowColors.Green).ToString(),  (int)ArrowColors.Green },
+        { ((int)ArrowColors.Red).ToString() +    ((int)ArrowColors.Red).ToString(),    (int)ArrowColors.Red },
+        { ((int)ArrowColors.Yellow).ToString() + ((int)ArrowColors.Yellow).ToString(), (int)ArrowColors.Yellow },
+        { ((int)ArrowColors.Blue).ToString() +   ((int)ArrowColors.Green).ToString(),  (int)ArrowColors.Cyan },
+        { ((int)ArrowColors.Green).ToString() +  ((int)ArrowColors.Red).ToString(),    (int)ArrowColors.GoldYellow },
+        { ((int)ArrowColors.Blue).ToString() +   ((int)ArrowColors.Red).ToString(),    (int)ArrowColors.Magenta },
+        { ((int)ArrowColors.Red).ToString() +    ((int)ArrowColors.Yellow).ToString(), (int)ArrowColors.Orange },
+        { ((int)ArrowColors.Blue).ToString() +   ((int)ArrowColors.Yellow).ToString(), (int)ArrowColors.Grey },
+        { ((int)ArrowColors.Green).ToString() +  ((int)ArrowColors.Yellow).ToString(), (int)ArrowColors.White },
+    };
+    // These dictionaries are intended to convert unfriendly enum names into friendly names for helpful logging
+    public static readonly Dictionary<PianoKeys, string> PianoKeyNames = new Dictionary<PianoKeys, string>
+    {
+        { PianoKeys.C, "C" },
+        { PianoKeys.Db, "C#/Db" },
+        { PianoKeys.D, "D" },
+        { PianoKeys.Eb, "D#/Eb" },
+        { PianoKeys.E, "E" },
+        { PianoKeys.F, "F" },
+        { PianoKeys.Gb, "F#/Gb" },
+        { PianoKeys.G, "G" },
+        { PianoKeys.Ab, "G#/Ab" },
+        { PianoKeys.A, "A" },
+        { PianoKeys.Bb, "A#/Bb" },
+        { PianoKeys.B, "B" },
```

Comment:
Have you tried using the actual characters ♯ and ♭ ? Do they pop up correctly in the logs?